index.html

```diff
@@ -57,10 +57,26 @@ <h1>Philip Gilbert</h1>
 
 
     <!-- about me: start -->
-    <section>
-
-       ABOUT ME CONTENT
-
+    <section class="container about-me">
+        <div class="row">
+            <img src="/img/About me/about-img.png.webp" alt="About me image" class="col-12 col-md-6 about-me-img">
+            <div>
+                <div class="col-12 col-md-6 about-me-title">
```

Comment:
is cia responsive klases perkelti vienu div auksciau ir atrodo normaliai

Reply:
pakeliau